post_installation_configuration/cluster-tasks.adoc

```diff
@@ -94,3 +94,25 @@ Understand and configure Pod disruption budgets.
 
 include::modules/nodes-pods-pod-disruption-about.adoc[leveloffset=+2]
 include::modules/nodes-pods-pod-disruption-configuring.adoc[leveloffset=+2]
+
+[id="post-install-rotate-cloud-creds"]
+== Rotating cloud provider credentials
+Some organizations require rotating the cloud provider credentials that were used to install {product-title} after installation. The xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator (CCO)] manages cloud provider credentials and can be configured to operate in several different modes. The process for rotating cloud credentials depends on the mode that the CCO is configured to use.
+
+[id="cred-rotation-cco-mode-mint"]
+=== Rotating credentials with the CCO in mint mode
+When the CCO is configured to operate in _mint mode_, it uses the provided admin-level cloud credential to create new credentials for each component with only the specific permissions that are required. Rotating credentials in mint mode requires the manual removal of the component credentials that were created by the removed credential.
+
+. Update the admin credentials root secret for your cloud provider.
+
+. Look up every credential that has been minted for the `credentialsRequest` CR and delete it from the cloud provider.
+
+. Delete each component secret that is referenced by the individual `credentialsRequest` objects.
```

Comment:
Can capitalize this as `CredentialsRequest` objects. They are found in the openshift-cloud-credential-operator namespace as well.

If it's helpful I noticed QE has this in their test plan for locating them:

```bash
$ oc -n openshift-cloud-credential-operator get CredentialsRequest -o json | jq -r '.items[] | select (.spec[].kind=="AWSProviderSpec") | .spec.secretRef'
```

Reply:
Yikes, I probably have this wrong elsewhere as well. Will fix here and make a note to self to find the others. Thanks!